### PR TITLE
Creating my post for my final app

### PR DESCRIPTION
**Pull request by ColinRNickels**
But I still need reflection which I am working on in another text editor and will paste here.

### CODE REVIEW

**Comment by ColinRNickels** [comment]
All done.
Ready to :goat: (my new favorite pun)

File: _posts/wookiemage/2016-04-28-Colins-Final-Project.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Looking ahead
+
+A number of us in this class want to keep going. We’ve made a lot of progress in this class but feel there is still so much we don’t understand. As far as we know, there is no second level python class, so we are looking for other ways to go forward. One such option is INLS 690-224 Visual Analytics; another one is INLS 690-163W Information Analytics. Both these courses rely on a good understanding of programming and will likely help push us further. I’m looking forward to taking these skills that we’ve garnered in this class and applying them to bigger data sets to create good analytics and visualizations.
```
Comment (eah13) [inline _posts/wookiemage/2016-04-28-Colins-Final-Project.md:29]: Check out the classes that Rob Capra and Jaime Arguello teach- both use or have opportunities to use Python more deeply.  You also can now look outside the department at CS classes like Bioalgortihms that will let you use Python within other domains.

File: _posts/wookiemage/2016-04-28-Colins-Final-Project.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Resources I’ve found
+
+I’ve found a lot of useful resources throughout this semester. Here are some for regexs:
+[pyregex]( http://www.pyregex.com/) This site is really useful for developing and testing a regex. It allows you to paste in your sample text and enter some regex. It will highlight any matches dynamically. Lots of sites do this; what makes this site special is the below cheat sheet.
+
+[Regex Crossword](https://regexcrossword.com/) Here’s one that I don’t understand at all. The goal is to take the obscurity of regexs and turn it into a game. There are lots of levels that use different features of regexs. It really tests your knowledge.
+
+Some Community Resources:
+
+(Learn Python Reddit)[https://www.reddit.com/r/learnpython/] This subreddit has over 50k subscribers of differing ability in Python who all have the same goal, to help others learn python. While I haven’t posted any questions to this subreddit, I have read a lot of great questions and explanations there. As with any subreddit, you get the bad with the good, so sometimes you see very terse answers with little explanation, but generally those get downvoted and nice verbose answers get upvoted.
```
Comment (eah13) [inline _posts/wookiemage/2016-04-28-Colins-Final-Project.md:44]: This link and the one below have the () and [] switched :)

File: _posts/wookiemage/2016-04-28-Colins-Final-Project.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+One thing that I would like would be a list of further resources and project ideas to continue to push ourselves further. I feel like a have a good grasp of the basics of python, but I’m not really sure where to go next. I know this is a common feeling among new programmers, as there are lots and lots of threads about this exact topic on the learn python subreddit. The answer always seems to be a variation of “keep going” or “make something for yourself.” I think a lot of people find this answer unsatisfying. Maybe we can make some sort of list of project ideas as a community effort. Or, more likely, such a thing already exists and I haven’t run into it yet. In fact, one small google search later, I found [this](https://www.reddit.com/r/dailyprogrammer). I haven’t had a chance to assess whether these challenges are appropriate, but they're a place to start.
```
Comment (eah13) [inline _posts/wookiemage/2016-04-28-Colins-Final-Project.md:54]: Great idea!  I'll add a final assignment, Keep Going with some resources that others can add to.  When I do, consider adding the links you shared here to it (it'll be in /_posts/exercises)

**Comment by eah13** [comment]
I'll send out more personalized feedback and a grade to you directly, but wanted to get a few comments to you now.  Congrats on being the first one done! 🚀

File: _posts/wookiemage/2016-04-28-Colins-Final-Project.md (hunk 3 above)
Comment (eah13) [inline _posts/wookiemage/2016-04-28-Colins-Final-Project.md:54]: (It's in there now)

**Pull request merged**